src/watertap_contrib/seto/multiperiod/VAGMD_batch_design_model.py

```diff
@@ -0,0 +1,386 @@
+    ]
+    PFluxAS7_high = [
+        9.41014468300000,
+        0,
+        0,
+        0,
+        -0.0188989390000000,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+    ]
+    PFluxAS7_low = [
+        4.82699491400000,
+        1.37479276300000,
+        1.91988177200000,
+        -0.574212905000000,
+        -0.641257664000000,
+        0.399259954000000,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        -0.588924321000000,
+        0,
+        0,
+    ]
+    TCOAS26 = [
+        65.1084685465240,
+        9.15474718837607,
+        -0.917460918908258,
+        0.480070517276181,
+        -1.06168979823129,
+        0,
+        0,
+        0,
+        0.142552983811052,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+    ]
+    TCOAS7_high = [
+        67.0068599900000,
+        0,
+        0,
+        0,
+        -0.0145469190000000,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+    ]
+    TCOAS7_low = [
+        58.8189910000000,
+        8.79691366400000,
+        -2.06741401400000,
+        1.63187967600000,
+        -0.914624645000000,
+        -0.536574144000000,
+        -0.249657477000000,
+        0.398973861000000,
+        -0.153760262000000,
+        0.102355281000000,
+        0.696768080000000,
+        -0.300582958000000,
+        -0.557410173000000,
+    ]
+    TEOAS26 = [
+        29.2261439896435,
+        0.569016152083381,
+        0.824636694807529,
+        4.62669502530487,
+        1.37222105534565,
+        0,
+        0,
+        0.220665657590258,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+    ]
+    TEOAS7_high = [
+        36.2497021800000,
+        0,
+        0,
+        0,
+        0.0126951860000000,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+    ]
+    TEOAS7_low = [
+        34.4309511000000,
+        1.55140768500000,
+        1.85928314600000,
+        4.52887180500000,
+        1.10791196800000,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0.808447211000000,
+    ]
+    Coder = [
+        [-5.68487382500000, 0.0705622560000000, 0.000152146000000000],
+        [-1.58460599600000, 0.00102338700000000, 1.20000000000000e-06],
+        [-4.27697973100000, 0.175533630000000, -0.000178178000000000],
+        [-1.49331349500000, 0.0146627780000000, 5.62000000000000e-06],
+        [-7, 0.100000000000000, 0],
+        [-2.14285714285714, 0.00285714285714286, 0],
+        [-5, 0.200000000000000, 0],
+        [-1.33333333333333, 0.00950000000000000, 0],
+    ]
+    a1 = 0.983930048493388
+    a2 = -4.8359231959954e-04
+    S_c = a1 * SgL + a2 * SgL**2  # [g/kg]
+
+    # TEI -= 273.15
+    # TCI -= 273.15
+
+    CoderVars = [
```

Comment:
We should have a description of what these variables represent somewhere would be nice. Don't need to rename the variables themselves (though that's preferred), but a comment in a docstring or something somewhere.

Reply:
Variable names updated.